neusomatic/python/extend_features.py

```diff
@@ -330,7 +330,7 @@ def extend_features(candidates_vcf,
     n_variants = len(all_variants)
     logger.info("Number of variants: {}".format(n_variants))
     split_len = (n_variants + num_threads - 1) // num_threads
-    pool = multiprocessing.Pool(num_threads)
+    # pool = multiprocessing.Pool(num_threads)
```

Comment:
@pavanstrand we should remove such commented out code, in general.

Reply:
Removed the commented code in all the files